### DISCUSSION TITLE
Graphic glitch.

### DISCUSSION THREAD

**Discussion by GoorooGaming** [freeplane/freeplane#1755] [Help]
Anyone have an idea why this is happening? I don't really know how to describe it. Took a video.
Win 11.

https://github.com/freeplane/docs/assets/164866813/ed185ddb-f383-45e1-861f-c8eb3c979041

**Comment by dpolivaev** [comment]
It happens on some machines I do not know why and when. It might be related to multiple monitors but it is also not certain.

**Comment by euu2021** [comment]
I added a comment at the main issue: https://github.com/freeplane/freeplane/issues/1348
